README.md

```diff
@@ -1,33 +1,26 @@
-# OmniAuth Auth0
+![Omniauth-auth0](https://cdn.auth0.com/website/sdks/banners/omniauth-auth0-banner.png)
 
 An [OmniAuth](https://github.com/intridea/omniauth) strategy for authenticating with [Auth0](https://auth0.com). This strategy is based on the [OmniAuth OAuth2](https://github.com/omniauth/omniauth-oauth2) strategy.
```

Comment:
I guess we can drop this one-liner based on the design


```suggestion
```

![image](https://user-images.githubusercontent.com/15910425/197721132-523d6d20-4609-4b6c-a0af-18066b741ae9.png)